### PR TITLE
Improve documentation of config parameters (GSI-463)

### PR DESCRIPTION
**Pull request by KerstenBreuer**
```
Make clear that the file ingest endpoint and the corresponding public key are coming from GHGA Central.
```

This contributes to the answer of https://github.com/ghga-de/forum/discussions/8.

### CODE REVIEW

File: ghga_datasteward_kit/s3_upload/config.py [hunk 1]
```diff
@@ -40,22 +40,41 @@ class LegacyConfig(BaseSettings):
...
+            "Access key ID for the local data hub's S3 server. The corresponding"
+            + " account need to have s3:GetObject and s3:PutObject privileges for the"
+            + " internal staging bucket (as in `bucket_id`). These credentials should"
+            + " never be shared with GHGA Central."
+        ),
     )
     s3_secret_access_key: SecretStr = Field(
-        ..., description="Secret access key for the S3 server"
+        ...,
+        description=("Secret access key corresponding to the `s3_access_key_id`."),
```
Comment (mephenor) [inline ghga_datasteward_kit/s3_upload/config.py:57]: This one is still a bit noninformative and might be confusing. Probably better to clarify that there's always a pair of `access_key_id` and `secret_access_key` needed to access S3, then this makes more sense to someone never confronted with the concept

**Review by mephenor — CHANGES_REQUESTED** [review]

File: ghga_datasteward_kit/s3_upload/config.py (hunk 1 above)
Comment (mephenor) [reply ghga_datasteward_kit/s3_upload/config.py:57]: Might be part of the long description of the access key ID, then this can be left as is.

Comment (KerstenBreuer) [reply ghga_datasteward_kit/s3_upload/config.py:57]: Good suggestion.

Comment (KerstenBreuer) [reply ghga_datasteward_kit/s3_upload/config.py:58]: Done

**Pull request merged**